lib/compat/wordpress-5.9/class-wp-theme-json-5-9.php

```diff
@@ -953,12 +954,13 @@ protected static function compute_preset_classes( $settings, $selector, $origins
 				foreach ( $slugs as $slug ) {
 					$css_var     = static::replace_slug_in_string( $preset_metadata['css_vars'], $slug );
 					$class_name  = static::replace_slug_in_string( $class, $slug );
+					$prioritise_user_presets = ( WP_Theme_JSON_Resolver_Gutenberg::theme_has_support() && true === $settings['prioritiseUserPresets'] ) ? '!important' : '';
```

Comment:
Do we need an `isset( $settings['prioritiseUserPresets'] )` check here?

Reply:
Yeh, and ff we do go ahead with this it may be that we are actually supposed to use the `maybe_opt_in_into_settings` method to set this as `appearanceTools` does, but didn't think I would waste too much time exploring that in case this doesn't fly 😄